packages/lesswrong/components/hooks/useGetParameter.tsx

```diff
@@ -0,0 +1,8 @@
+export const useGetParameter = (name: string) => {
+  for (const token of location.search.substr(1).split("&")) {
+    const [key, value] = token.split("=");
+    if (key === name) {
+      return value;
+    }
+  }
```

Comment:
Any way we can use native `URL` functionality, i.e. by making a new URL and then using `searchParams.get(name)`?  Seems like it'd be more readable and less error-prone

Reply:
Good point - I forgot `URLSearchParams` is a thing!